main.go

```
@@ -0,0 +1,155 @@
+				// 		logger.Errorf("failed get file from bot api: %v", err)
+				// 		response.Text = "failed store document"
+				// 	} else {
+				// 		response.Text = "success"
+				// 	}
+				// 	bot.Send(response)
+				// }
+			}
+		}
+	}
+
+	eg.Wait()
```

golangci-lint (1.62.2)

Error return value of `eg.Wait` is not checked

(errcheck)